frontend/src/components/Board/RegularBoard/ParticipantsList/ParticipantCard.tsx/index.tsx

```diff
@@ -11,91 +12,156 @@ import { BoardUser, BoardUserAddAndRemove } from '@/types/board/board.user';
-      addBoardUsers: [],
-      removeBoardUsers: [member._id],
-      boardId,
+    const handleRemove = () => {
+      if (!member._id) return;
+
+      const boardUsersToUpdate: BoardUserAddAndRemove = {
+        addBoardUsers: [],
+        removeBoardUsers: [member._id],
+        boardId,
+      };
+
+      mutate(boardUsersToUpdate);
     };
 
-    mutate(boardUsersToUpdate);
-  };
-  return (
-    <Flex css={{ flex: '1 1 1' }} direction="column">
-      <Flex>
-        <InnerContainer
-          align="center"
-          elevation="1"
-          justify="between"
-          css={{
-            position: 'relative',
-            flex: '1 1 0',
-            py: '$22',
-            maxHeight: '$76',
-            ml: 0,
-          }}
-        >
-          <Flex align="center" css={{ width: '100%' }} gap="8">
-            <Icon
-              name="blob-personal"
-              css={{
-                width: '32px',
-                height: '$32',
-                zIndex: 1,
-                opacity: isOpen ? 0.2 : 1,
-              }}
-            />
-            <Flex align="center" gap="8" justify="start" css={{ width: '50%' }}>
-              <StyledMemberTitle>
-                {`${member.user.firstName} ${member.user.lastName}`}
-              </StyledMemberTitle>
-            </Flex>
-            {!isBoardCreator && (
-              <Flex align="center" css={{ width: '50%' }} justify="end">
-                <Tooltip content="Remove participant">
-                  <Flex justify="end">
-                    <IconButton
-                      onClick={handleRemove}
-                      css={{
-                        '&:hover': {
-                          cursor: 'pointer',
-                        },
-                      }}
-                    >
-                      <Icon
-                        name="trash-alt"
-                        css={{
-                          color: '$primary400',
-                          size: '$20',
-                        }}
-                      />
-                    </IconButton>
+    const updateIsResponsibleStatus = (checked: boolean) => {
+      // if (member.team) {
+      //   const updateTeamUser: TeamUserUpdate = {
+      //     team: member.team,
+      //     user: member.user._id,
+      //     role: member.role,
+      //     isNewJoiner: checked,
+      //   };
+
+      //   mutate(updateTeamUser);
+      // }
+      const participantsList = [...boardParticipants];
+      const idxParticipantToUpdate = participantsList.findIndex(
+        (boardUser) => boardUser._id === member._id,
+      );
+      participantsList.splice(idxParticipantToUpdate, 1);
+      if (checked) {
+        participantsList.splice(idxParticipantToUpdate, 0, {
+          ...member,
+          role: BoardUserRoles.RESPONSIBLE,
+        });
+      } else {
+        participantsList.splice(idxParticipantToUpdate, 0, {
+          ...member,
+          role: BoardUserRoles.MEMBER,
+        });
+      }
```

Comment:
```suggestion
      participantsList.splice(idxParticipantToUpdate, 0, {
           ...member,
           role: checked  ? BoardUserRoles.RESPONSIBLE :  BoardUserRoles.MEMBER,
         });
```BoardUserRoles.MEMB if (checked) {
        participantsList.splice(idxParticipantToUpdate, 0, {
          ...member,
          role: BoardUserRoles.RESPONSIBLE,
        });
      } else {
        participantsList.splice(idxParticipantToUpdate, 0, {
          ...member,
          role: BoardUserRoles.MEMBER,
        });
      }